python/tests/test_sbml_import.py

```diff
@@ -115,10 +115,7 @@ def test_nosensi(tempdir):
 
     model = model_module.getModel()
     model.setTimepoints(np.linspace(0, 60, 61))
-    solver = model.getSolver()
-    solver.setSensitivityOrder(amici.SensitivityOrder.first)
-    solver.setSensitivityMethod(amici.SensitivityMethod.forward)
-    rdata = amici.runAmiciSimulation(model, solver)
+    rdata = model.simulate(sensi_order="first", sensi_method="forward")
```

Comment:
```suggestion
    rdata = model.simulate(sensi_order=amici.SensitivityOrder.first, sensi_method=amici.SensitivityMethod.forward)
```

Reply:
Both are supported. I like the former, because it's less verbose, in particular for interactive use.